### DISCUSSION TITLE
python/05.python-basis-loop

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#24] [General]
# python/05.python-basis-loop

https://docs.geeksman.com/python/05.python-basis-loop.html

**Comment by xujiangyu0619** [comment]
博主，您好
字典实现其推导式语法的示例中有误，
info_dict = {key, value for key, value in zip(keys, values)}
应该是
info_dict = {key:value for key, value in zip(keys, values)}

**Reply by Pavel-Shcherbo** [reply]
Response #1

**Comment by Pavel-Shcherbo** [comment]
Response #2
